src/testFixtures/kotlin/artwork/ArtworkUtils.kt

```diff
@@ -0,0 +1,39 @@
+package artwork
+
+import getRandomObjectId
+import ua.marchenko.artauction.artwork.controller.dto.CreateArtworkRequest
+import ua.marchenko.artauction.artwork.enums.ArtworkStatus
+import ua.marchenko.artauction.artwork.enums.ArtworkStyle
+import ua.marchenko.artauction.artwork.model.Artwork
+import getRandomString
+import ua.marchenko.artauction.common.mongodb.id.toObjectId
+import user.getRandomUser
+import ua.marchenko.artauction.user.enums.Role
+import ua.marchenko.artauction.user.model.User
+
+fun getRandomArtwork(
```

Comment:
as another way, you can make extension like
Artwork.Companion.random...
and use it 
Artwork.random(...)

I don't force you to rewrite this code, just good to know

Reply:
Do it good practice to rewrite random function for Strings and ObjectId too?
Like rewrite `fun getRandomString(length: Int = 10): String` to `fun String.Companion.random(length: Int = 10): String`?